source/parser.py

```diff
@@ -242,51 +423,47 @@ def main():
-                    print(f'Almost same titles: {grobid.title}')
-                    paper_title = g_title2
-                elif g_title.lower() in c_title.lower() or grobid.title.lower() in cermine.title.lower() or g_title2.lower() in c_title2.lower():
-                    print(f'Partial title: {grobid.title} is part of {cermine.title}')
-                    paper_title =  grobid.title
-                elif c_title.lower() in g_title.lower() or cermine.title.lower() in grobid.title.lower() or c_title2.lower() in g_title2.lower():
-                    print(f'Partial title: {cermine.title} is part of {grobid.title}')
-                    paper_title = cermine.title
+                url_path = paper_path + ".pdf"
+
+                #check it is preface or paper
+                if not p == "Preface":
+                    # extract metadata for each paper using GROBID 
+                    grobid =  GrobitFile(paper_path + '.grobid')
+
+                    # extract metadata for each paper using CERMINE 
+                    cermine =  CermineFile(paper_path + '.cermine')
+                    print(k, p)
+
+                    #compare the title and give the final result
+                    final_title = compare_title(grobid, cermine, url_path)
+                    
+                    #merge author information 
+                    final_author_list = compare_author(grobid, cermine)
+                    final_author_list = grobid.authors
+
+                            
+                    # Create Neo4j graph based on extracted metadata
+                    create_neo4j_graph(final_author_list,final_title, neo4j_conn, url_path) 
+                                 
+                #if it is paper, it should belongs to one event
                 else:
-                    #check if string similarity is above a threshold ussing fuzzy matching
-                    if fuzz.ratio(cermine.title, grobid.title) > 95:
-                        paper_title = cermine.title
-                    else :
-                        # TODO: need to decide what to do here
-                        print('Titles not the same: COME UP with solution on how to resolve the conflicts')
-                        print(cermine.title, '\n', grobid.title, '\n')
-
-                #merge author information 
+                    # extract metadata for each paper using GROBID 
+                    grobid =  GrobitFile(paper_path + '.grobid')
+
+                    # extract metadata for each paper using CERMINE 
+                    cermine =  CermineFile(paper_path + '.cermine')
+                    print(k, p)
+
+                    #compare the title and give the final result
+                    final_title = compare_title(grobid, cermine, url_path)
+                    
+                    #merge author information 
+                    final_author_list = compare_author(grobid, cermine)
+                    final_author_list = grobid.authors
```

Comment:
did you add this line because the results from the author comparison method were not as expected?